app/models/experiment.rb

```diff
@@ -762,7 +762,7 @@ def update_module_groups(module_groups, current_user)
     dg.to_undirected.each_connected_component { |w| workflows <<  w }
 
     # Retrieve maximum allowed module group name
-    max_length = (MyModuleGroup.validators_on(:name).select { |v| v.class == ActiveModel::Validations::LengthValidator }).first.options[:maximum]
+    max_length = Constants::NAME_MAX_LENGTH
```

Comment:
It would be nicer to remove this line and use `Constants::NAME_MAX_LENGTH` directly instead, just as it is used on line `367`.